code_1.rb

```diff
@@ -0,0 +1,104 @@
+class Station # класс станция
+  attr_reader :trains, :name_station # возвращает список поездов, находящися на станции
+  def initialize(name_station) # имя станции указывается при создании
+    @name_station = name_station
+    @trains = [] # массив поездов
+  end
+  def add_trains (train) # принимает поезда по одному за раз
```

Comment:
не оставляй пробел между именем метода и скобками - это может привести к ошибкам в работе интерпретатора руби
```suggestion
  def add_trains(train) # принимает поезда по одному за раз
```